thunder/torch/__init__.py

```diff
@@ -974,6 +974,13 @@ def view(a: TensorLike, /, *shape) -> TensorLike:
     return reshape(a, shape)
 
 
+@torchsymbol(torch.broadcast_tensors, is_method=True)
+def broadcast_tensors(*inputs):
+    if len(inputs) == 1 and not isinstance(inputs[0], TensorProxy):
```

Comment:
What's going on with this check?

Reply:
I think I should remove this.
When referring to [this](https://github.com/pytorch/pytorch/blob/ea61c9cb299b6dfebc57dc9d8821c34321d568ab/torch/_inductor/decomposition.py#L661), I thought there can be a use case when input and target tensors are passed in a list. However, when trying the following, I guess `torch.broadcast_tensors` does not support that. I should have checked this earlier. Sorry for the confusion!
```python
x = torch.randn(1, 2, 3)
y = torch.randn(4, 5, 6)
a, b = torch.broadcast_tensors([x, y])
```
```
  File "/opt/homebrew/anaconda3/lib/python3.11/site-packages/torch/functional.py", line 76, in broadcast_tensors
    return _VF.broadcast_tensors(tensors)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected Tensor as element 0 in argument 0, but got list
```

Reply:
Removing this sounds good. You're right I don't think we or PyTorch support passing tensors in a list. Although someone could do `*lst` if they wanted to pass the tensors in a list to the operation